translator/trace/jaeger/protospan_to_jaegerproto.go

```diff
@@ -530,3 +530,10 @@ func ocSpansToJaegerSpansProto(ocSpans []*tracepb.Span) ([]*jaeger.Span, error)
 
 	return jSpans, nil
 }
+
+func truncatableStringToStr(ts *tracepb.TruncatableString) string {
```

Comment:
I have a slight preference to use `GetValue()` from `TruncatableString` instead of adding a function that is equivalent to it. That said some prefer to see the nil check directly on our sources, up to you.

Reply:
In the deleted file there was a function `truncableStringToStr` (I fixed the name only) and I tried to not change too many things. Happy to review a PR after.